lib/hyrax/publisher.rb

```diff
@@ -96,6 +96,12 @@ class Publisher
 
     # @since 3.0.0
     # @macro a_registered_event
+    #   @note this event SHOULD be published whevener the metadata is saved
+    #     for a PCDM Collection. the payload for each published event MUST
+    #     include an `:collection` (the updated Collection) AND a `:user` (the
```

Comment:
afaict this event and `object.metadata.updated` both ignore the user field https://github.com/samvera/hyrax/blob/main/app/services/hyrax/listeners/metadata_index_listener.rb#L20-L34

Reply:
other listeners (e.g. `ObjectLifecycleListener`) require this data.

recall that any given channel/topic can have many listeners.  once we are publishing data in a payload in a released version of the engine, removing that data is a high bar, since we don't control downstream listener behavior.

Reply:
oh, right right